libs/shell/presentational/src/lib/ui/gor-data-formatter/gor-data-formatter.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'gorilainvest-gor-data-formatter',
+  templateUrl: './gor-data-formatter.component.html',
+  styleUrls: ['./gor-data-formatter.component.scss']
+})
+export class GorDataFormatterComponent implements OnInit {
+  dataTypeFormControl = new FormControl('date');
+  valueFormControl = new FormControl('2019-01-02');
+  dataOptionFormControl = new FormControl('1.2-2');
+  dataStyleFormControl = new FormControl('');
+  displayDataOption = false;
+  isCustom = false;
+
+  constructor() {
+    this.dataTypeFormControl.valueChanges.subscribe(data => this.checkType(data));
```

Comment:
Is a bad practice to do subscribes on class constructors, do it in another place instead

Reply:
ok